linkerd/app/outbound/src/http/logical/policy/route/retry.rs

```diff
@@ -29,6 +31,9 @@ pub struct RetryPolicy {
     pub backoff: Option<ExponentialBackoff>,
 }
 
+#[derive(Clone, Debug)]
+pub struct RetryLabelExtract(pub ParentRef, pub RouteRef);
```

Comment:
our new extractor, which hangs onto the matched route's parent and route metadata.